docs/getting-started/example-bpf.md

```diff
@@ -189,76 +186,62 @@ _Other program types are the same._
+[Building Locally](#building-locally) section).
 
-```console
-cd bpfman/examples/go-xdp-counter/
-go generate
+To generate the bytecode for all the examples:
 
-docker build \
-  --build-arg PROGRAM_NAME=go-xdp-counter \
-  --build-arg BPF_FUNCTION_NAME=xdp_stats \
-  --build-arg PROGRAM_TYPE=xdp \
-  --build-arg BYTECODE_FILENAME=bpf_bpfel.o \
-  --build-arg KERNEL_COMPILE_VER=$(uname -r) \
-  -f ../../Containerfile.bytecode . -t quay.io/$USER/go-xdp-counter-bytecode:latest
+```console
+cd bpfman/examples/
+make generate
 ```
 
-and
+OR to generate the bytecode for a single example (XDP in this case):
 
 ```console
-cd bpfman/examples/go-tc-counter/
+cd bpfman/examples/go-xdp-counter/
 go generate
+```
+
+The preferred method for building the container image is to use the `bpfman image build`
+command.
+See [bpfman image build](./cli-guide.md#bpfman-image-build) in the CLI Guide for more details.
 
-docker build \
-  --build-arg PROGRAM_NAME=go-tc-counter \
-  --build-arg BPF_FUNCTION_NAME=stats \
-  --build-arg PROGRAM_TYPE=tc \
-  --build-arg BYTECODE_FILENAME=bpf_bpfel.o \
-  --build-arg KERNEL_COMPILE_VER=$(uname -r) \
-  -f ../../Containerfile.bytecode . -t quay.io/$USER/go-tc-counter-bytecode:latest
+```console
+cd bpfman/examples/go-xdp-counter/
+bpfman image build -f ../../Containerfile.bytecode -t quay.io/$QUAY_USER/go-xdp-counter-bytecode:test -b bpf_x86_bpfel.o
 ```
 
-_Other program types are the same._
+The examples [Makefile](https://github.com/bpfman/bpfman/blob/main/examples/Makefile) has
+commands to build all the example images if needed.
+See [Locally Build Example Container Images](../developer-guide/image-build.md#locally-build-example-container-images)
+for more details.
 
 `bpfman` currently does not provide a method for pre-loading bytecode images
-(see [issue #603](https://github.com/bpfman/bpfman/issues/603)), so push the bytecode image to a remote
+(see [issue #603](https://github.com/bpfman/bpfman/issues/603)), so push the bytecode image to an image
 repository.
+
 For example:
 
 ```console
 docker login quay.io
-docker push quay.io/$USER/go-xdp-counter-bytecode:latest
-docker push quay.io/$USER/go-tc-counter-bytecode:latest
-```
-
-Then run with the privately built bytecode container image:
-
-```console
-sudo ./go-tc-counter -iface ens3 -direction ingress -image quay.io/$USER/go-tc-counter-bytecode:latest
-2022/12/02 16:38:44 Using Input: Interface=ens3 Priority=50 Source=quay.io/$USER/go-tc-counter-bytecode:latest
-2022/12/02 16:38:45 Program registered with id 6225
-2022/12/02 16:38:48 4 packets received
-2022/12/02 16:38:48 580 bytes received
-
-2022/12/02 16:38:51 4 packets received
-2022/12/02 16:38:51 580 bytes received
-
-^C2022/12/02 16:38:51 Exiting...
-2022/12/02 16:38:51 Unloading Program: 6225
+docker push quay.io/$QUAY_USER/go-xdp-counter-bytecode:test
 ```
 
 ## Running Examples
 
+Below are some examples of how to run the examples on a host where bpfman is already installed.
```

Comment:
"examples of how to run the examples" sounds redundant.  The following might be a little better.

```suggestion
Below are some examples of how to run the bpfman examples on a host where bpfman is already installed.
```

Reply:
Done.